### ISSUE TITLE
Elbow contracture

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7689]
**Preferred term label:**
Elbow contracture

**Synonyms**

**Definition (free text, please give PubMed ID)**
A limitation in the passive range of motion of the elbow due to shortening of the periarticular connective tissues and muscles, preventing the normal range of movement of the elbow

**Parent term (use hpo.jax.org/app)**
Limited elbow movement HP:0002996
and
Contractures of the joints of the upper limbs HP:0100360  (Note we should rename this to Upper-limb joint contracture)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Suggestion by Harold J.P. van Bosse
Elbow contracture: this definition is good, but only part of the story. Contractures may also occur due to internal issues of the joint, both soft and hard tissues. Ankylosis of a joint often includes fibrosis of the joint (essentially scar tissue developing with the joint, inhibiting the bony articular surfaces from moving against eachother), as in a frozen shoulder, for example. In the knee, as another example, you can have a lack of motion due to a meniscal tear (that is intra articular, not peri-articular), or due to an osteochondritis dissecans lesion (articular cartilage detachment). Also, arthritis of a joint will cause limitations of motion, described as contractures (think of hips and knees). All these examples also apply to contractures of the elbow joint, so they can be hard and soft tissue in origin, congenital or acquired, peri or intra articular in nature.

**Comment by pnrobinson** [comment]
Revised definition:
A limitation in the passive range of motion of the elbow resulting from loss of elasticity in the periarticular tissues owing to structural changes of non-bony tissues, such as muscles, tendons, ligaments, joint capsules or skin.

**Comment by pnrobinson** [comment]
added
